tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -16,4 +16,6 @@
     <!-- Eclipse SmartHome specific suppressions-->
     <!-- These bundles are generated trough XText -->
     <suppress files=".+org.eclipse.smarthome.model.+" checks="RequireBundleCheck|ExportInternalPackageCheck|ManifestPackageVersionCheck|ImportExportedPackagesCheck"/>
+    <!-- Some source files have different headers -->
+    <suppress files=".+org.eclipse.smarthome.automation.+" checks="HeaderCheck"/>
```

Comment:
Do we really want to suppress checks or rather update the automation bundles to adhere to our guidelines?